### DISCUSSION TITLE
fix-or-question: graphiql and `dev`

### DISCUSSION THREAD

**Discussion by jjangga0214** [graphql-hive/gateway#146] [Q&A] [answered]
**gateway.config.ts**

```ts
import { defineConfig } from '@graphql-hive/gateway'

export const gatewayConfig = defineConfig({
  port: 8080,
  graphiql: true,
  // ... other options are omitted for brevity
})

```

Let's say I execute this command.

```sh
$ hive dev \
        --service auth --url http://localhost:8081/graphql \
        --service api --url http://localhost:8083/graphql \
        --watch
```

Then a successful message is printed out.

```sh
ℹ Watch mode enabled
✔ Composition successful
Saving supergraph schema to supergraph.graphql
ℹ Watching for changes
```

But there's no response when I access `http://localhost:8080` or `http://localhost:8080/graphql`, etc.

I also wonder about the API endpoint. 
Where should I send a request like `POST` for query and mutation or websocket for subscription?
Is it configurable?
(As far as I searched, there's no info from the docs.)

Thanks.

**Comment by enisdenjo** [comment, marked as answer]
You're not running the Hive Gateway, only the Hive CLI. See ["Starting the Gateway" in documentation](https://the-guild.dev/graphql/hive/docs/gateway#starting-the-gateway).

The command for running the gateway is `hive-gateway`. I don't know your full config, but I assume it should be `hive-gateway supergraph`.

**Comment by jjangga0214** [comment]
Thank you!!
